VTEX - Payment Provider Protocol.json

```diff
@@ -1718,112 +2141,58 @@
-                            "example": "{{action}}"
-                        }
-                    },
-                    {
-                        "name": "X-VTEX-API-AppKey",
-                        "in": "header",
-                        "description": "The AppKey configured by the merchant (optional configuration)",
-                        "required": true,
-                        "style": "simple",
-                        "schema": {
-                            "type": "string",
-                            "default": "{{X-VTEX-API-AppKey}}"
-                        }
-                    },
-                    {
-                        "name": "X-VTEX-API-AppToken",
-                        "in": "header",
-                        "description": "The AppToken configured by the merchant (optional configuration)",
-                        "required": true,
-                        "style": "simple",
-                        "schema": {
-                            "type": "string",
-                            "default": "{{X-VTEX-API-AppToken}}"
-                        }
-                    },
-                    {
-                        "name": "Accept",
-                        "in": "header",
-                        "description": "Media type(s) that is/are acceptable for the response. Default value for payment provider protocol is application/json",
-                        "required": true,
-                        "style": "simple",
-                        "schema": {
-                            "type": "string",
-                            "default": "application/json"
-                        }
+                        "$ref": "#/components/parameters/Content-Type"
                     },
                     {
-                        "name": "Content-Type",
-                        "in": "header",
-                        "description": "The Media type of the body of the request. Default value for payment provider protocol is application/json",
-                        "required": true,
-                        "style": "simple",
-                        "schema": {
-                            "type": "string",
-                            "default": "application/json"
-                        }
+                        "$ref": "#/components/parameters/Accept"
+                    },
+                    {
+                        "$ref": "#/components/parameters/paymentId"
                     },
                     {
-                        "name": "paymentId",
+                        "name": "action",
                         "in": "path",
-                        "description": "VTEX payment identifier",
+                        "description": "It is the same `:action` information used in the `inboundRequestsUrl` provided in the [Create Payment](https://developers.vtex.com/docs/api-reference/payment-provider-protocol#post-/payments?endpoint=post-/payments) endpoint payload.",
```

Comment:
I didn't understand the `action` parameter. What are possible values? What are the impacts of sending a given `action` in your request? Maybe examples help

Reply:
Adicionado exemplos de hook e enrollment.